pmlc/conversion/stdx_to_llvm/stdx_to_llvm.cc

```diff
@@ -70,6 +70,18 @@ class LLVMLegalizationPattern : public ConvertToLLVMPattern {
   LLVM::LLVMDialect &dialect;
 };
 
+LLVM::LLVMFuncOp appendOrGetFuncOp(StringRef funcName, LLVM::LLVMType funcType,
```

Comment:
I think we've used `getOrInsertFuncOp` in other places, let's keep it consistent.